charts/prometheus-kafka-exporter/Chart.yaml

```
@@ -3,7 +3,7 @@ appVersion: "v1.9.0"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 2.12.2
+version: 2.12.3
```

A new feature is a minor change:

```suggestion
version: 2.13.0
```